src/components/web-monitoring-ui.jsx

```diff
@@ -65,11 +65,22 @@ export default class WebMonitoringUi extends React.Component {
   logOut () {
     api.logOut();
     this.setState({user: api.userData});
-    this.loadPages('pages');
+    this.loadPages();
```

Comment:
I want to refactor this to call `loadPages` on `componentDidUpdate` but it after looking at the code, it is not as simple as just putting that function inside the lifecycle hook. Will probably take this on in a later PR unless you have concerns